src/UI/FlatList.re

```diff
@@ -82,23 +120,54 @@ let createElement =
       setScrollTop(newScrollTop);
     };
 
+    let scrollbar = {
+      let maxHeight =
+        (count * rowHeight) - menuHeight;
+      let thumbHeight =
+        menuHeight * menuHeight / max(1, (count * rowHeight));
+      let isVisible = 
+        maxHeight > 0;
+
+      if (isVisible) {
+        <View style=Styles.slider>
+          <Slider
+            onValueChanged={v => setScrollTop(int_of_float(v))}
+            minimumValue=0.
+            maximumValue={float_of_int(maxHeight)}
+            sliderLength=menuHeight
+            thumbLength=thumbHeight
+            value={float_of_int(actualScrollTop)}
+            trackThickness=Constants.scrollBarThickness
+            thumbThickness=Constants.scrollBarThickness
+            minimumTrackColor=Constants.scrollTrackColor
+            maximumTrackColor=Constants.scrollTrackColor
+            thumbColor=Constants.scrollThumbColor
+            vertical=true
+          />
+        </View>
+      } else {
+        React.empty
+      }
+    };
+
     let items = render(
       ~menuHeight,
       ~rowHeight,
       ~count,
       ~scrollTop = actualScrollTop,
       ~renderItem
     );
+    (
+      hooks,
+      <View
+        style=Styles.container(~width, ~height=min(menuHeight, count * rowHeight))
+        onMouseWheel=scroll >
+
+        <View style=Styles.viewport(~isScrollbarVisible=scrollbar==React.empty)>
+          ...items
+        </View>
```

Comment:
Wrapping items in another `View` cause `onMouseOver` to stop triggering. Seems like a bug in revery though.

Reply:
Interesting - I can follow up on this part 👍